master/internal/prom/utli_test.go

```diff
@@ -0,0 +1,43 @@
+package prom_test
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/determined-ai/determined/master/internal/prom"
+)
+
+var (
+	labels    = []string{"method"}
+	histogram = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: prom.DeterminedNamespace,
+		Subsystem: "my-subsystem",
+		Name:      "seconds",
+		Buckets:   prometheus.DefBuckets,
+	}, labels)
+)
+
+func ExampleTime() {
+	defer prom.Time(histogram.WithLabelValues("GET"))
+
+	// do thing you want to time.
+	time.Sleep(time.Millisecond)
+	// Output:
```

Comment:
is `ExampleTime` or `ExampleErrCount` invoked elsewhere, or automatically called? 

Also reminder to fill out the output comment, unless it's empty?

Reply:
This is a testable example (https://go.dev/blog/examples). It should get ran as part of go test. The output comment is an assertion of what the test should print.

Reply:
This way if you open the docs for our go package, there are nice docs on how to use this.

Reply:
I prob could actually assert something about the test 🤷 , but then I need to introspect the prom metrics.

Reply:
I see, that makes sense